app/assets/stylesheets/registrations.css

```diff
@@ -0,0 +1,61 @@
+.login-section, .signup-section {
+  text-align: center;
+}
+
+.login-title, .signup-title {
+  font-family: Abril Fatface;
+  color: #000000;
```

Comment:
You can use black directly. This are standard values for colors offered by css.
If you are using always the same colors, you can create variables for those.
http://sass-lang.com/guide#topic-2